xml/cap_depl_install_production.xml

```diff
@@ -41,7 +41,43 @@
   &productname; cluster, and not for &caasp; or &ses;. See
   <xref linkend="sec.cap.ha-prod"/>.
  </para>
- <sect1 xml:id="sec.cap.prereqs-prod">
+ 
+<sect1 xml:id="sec.cap.deploy-notes">
+  <title>Deployment and Administration Notes</title>
+  <para>
+      This section lists considerations for deploying and operating
+      &productname;.
+  </para>
+  
+  <variablelist>
+   <varlistentry>
+       <term>Length of release names limits</term>
```

Comment:
Would "Length of release names" or "Release name length" work here? Labeling it "limits" makes the product sounds so...limiting heh

Reply:
ee such clumsy wording :)